src/azure-cli-core/azure/cli/core/util.py

```diff
@@ -1077,6 +1077,22 @@ def get_az_user_agent():
     return ' '.join(agents)
 
 
+def get_az_rest_user_agent():
+    """Get User-Agent for az rest calls"""
+
+    from msrest import __version__ as msrest_version
+    from msrestazure import __version__ as msrestazure_version
+
+    agents = ['python/{}'.format(platform.python_version()),
+              '({})'.format(platform.platform()),
+              'msrest/{}'.format(msrest_version),
+              'msrest_azure/{}'.format(msrestazure_version),
```

Comment:
`az rest` never relies on `msrest` and `msrest_azure`. I doubt if we should add them.

Reply:
I agree, removed.